test/shopify-cli/commands/connect_test.rb

```diff
@@ -5,7 +5,80 @@ module Commands
     class ConnectTest < MiniTest::Test
       include TestHelpers::Partners
 
-      def test_run
+      def test_connect_asks_project_type_and_writes_yml_when_no_project_exists
+        ShopifyCli::Project.stubs(:has_current?).returns(false)
+        Resources::EnvFile.expects(:parse_external_env).returns({})
+        ShopifyCli::PartnersAPI::Organizations.stubs(:fetch_with_app).returns(partners_api_response)
+        CLI::UI::Prompt.expects(:ask).with("What type of project would you like to connect?").returns('node')
+        CLI::UI::Prompt.expects(:ask).with('To which organization does this project belong?').returns(101)
+        CLI::UI::Prompt.expects(:ask).with(
+          'Which development store would you like to use?'
+        ).returns('store.myshopify.com')
```

Comment:
Also minor, but if these messages change this test will start breaking. Should we call `message` here to get the expected texts?

Reply:
IMO it's fine this way. We'd be reducing test readability since there would be an extra later of indirection.

Reply:
That's fair. 2 things:
* I've seen other tests that actually use the keys rather than the translations. If we do decide to keep the texts we should consistently use either one or the other.
* I agree that it affects readability, and it only becomes a problem when the text changes, which is likely not often. However, tying the tests to translations seems a bit fragile to me. Just thinking aloud here, BTW - I could go either way on this one.

Reply:
If we use the keys already in tests then I'd support keeping it that way for consistency.

Reply:
+1 for using the key instead so that when strings are updated, it doesn't break tests.

Reply:
I've updated the tests for `connect` here and all the other tests in https://github.com/Shopify/shopify-app-cli/pull/687